service/history/events/cache.go

```diff
@@ -52,23 +53,23 @@ type (
 	}
 
 	Cache interface {
-		GetEvent(ctx context.Context, key EventKey, firstEventID int64, branchToken []byte) (*historypb.HistoryEvent, error)
+		GetEvent(ctx context.Context, shardID int32, key EventKey, firstEventID int64, branchToken []byte) (*historypb.HistoryEvent, error)
```

Comment:
ID is a short for Identifier. Should be Id (loweecase d)

Reply:
It is an identifier. We use shardID everywhere else in the code.